lib/cocoapods/resolver.rb

```diff
@@ -439,7 +439,7 @@ def handle_resolver_error(error)
               message << "\n\nNone of your spec sources contain a spec satisfying "\
                 "the #{dependencies}: `#{conflicts.join(', ')}`." \
                 "\n\nYou have either:" \
-                "\n * out-of-date source repos which you can update with `pod repo update`." \
+                "\n * out-of-date source repos, try to run `pod install --repo-update` which will update them." \
```

Comment:
Can we include the previous command but mention the `pod install --repo-update` option as an alternative?